cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -125,99 +148,23 @@ public static void main( String[] args )
      * @return the exit code.
      */
     @SuppressWarnings("WeakerAccess") // Public API.
-    public int run( String[] args )
-    {
-        Options options = new Options();
-        options.addOption( "h", false, "Print this help message." );
-        options.addOption( Option.builder( "d" ).longOpt( "debug" ).desc( "Enable debug" ).build() );
-        options.addOption( Option.builder( "t" ).longOpt( "trace" ).desc( "Enable trace" ).build() );
-        options.addOption( Option.builder( "h" ).longOpt( "help" ).desc( "Print help" ).build() );
-        options.addOption( Option.builder( "f" )
-                                 .longOpt( "file" )
-                                 .hasArgs()
-                                 .numberOfArgs( 1 )
-                                 .desc( "POM file" )
-                                 .build() );
-        options.addOption( Option.builder( "l" )
-                                 .longOpt( "log" )
-                                 .desc( "Log file to output logging to" )
-                                 .numberOfArgs( 1 )
-                                 .build() );
-        options.addOption( Option.builder( "s" )
-                                 .longOpt( "settings" )
-                                 .hasArgs()
-                                 .numberOfArgs( 1 )
-                                 .desc( "Optional settings.xml file" )
-                                 .build() );
-        options.addOption( Option.builder( "P" )
-                                 .longOpt( "activeProfiles" )
-                                 .desc( "Comma separated list of active profiles." )
-                                 .numberOfArgs( 1 )
-                                 .build() );
-        options.addOption( Option.builder( "o" )
-                                 .longOpt( "outputFile" )
-                                 .desc( "outputFile to output dependencies to. Only used with '-p' (Print all project dependencies)" )
-                                 .numberOfArgs( 1 )
-                                 .build() );
-        options.addOption( Option.builder( "p" ).longOpt( "printDeps" ).desc( "Print all project dependencies" ).build() );
```

Comment:
Seems you changed this option name, but you also missed adding short option `-p`.

Reply:
Yes :-)